examples/chart/teleport-agent/README.md

```diff
@@ -0,0 +1,123 @@
+# Teleport Agent chart
+
+This chart is a Teleport agent used to register any or all of the following services
+with an existing Teleport cluster:
+- Teleport Kubernetes access
+- Teleport Application access
+- Teleport Database access
+
+To use it, you will need:
+- an existing Teleport cluster (at least proxy and auth services)
+- a reachable proxy endpoint (`$PROXY_ENDPOINT`)
+- a [static join
+  token](https://goteleport.com/teleport/docs/admin-guide/#adding-nodes-to-the-cluster)
+  for this Teleport cluster (`$JOIN_TOKEN`)
+  - this chart does not currently support dynamic join tokens; please [file an
+    issue](https://github.com/gravitational/teleport/issues/new?labels=type%3A+feature+request&template=feature_request.md)
+    if you require support for dynamic tokens
+
+## Combining roles
+
+You can combine multiple roles as a comma-separated list: `--set roles=kube\,db\,app`
```

Comment:
For database access the role name is `database` as of 6.0.0-alpha.1. We have a ticket to rename it to `db` (https://github.com/gravitational/teleport/issues/5349) which I'll likely do soon, but for now `--set roles=kube\,database\,app`.

Reply:
FWIW I put out a [PR](https://github.com/gravitational/teleport/pull/5359) to rename it but if we're gonna deploy first alpha initially, we'll have to use database here now and rename after.

Reply:
I think I'll leave this as-is - the key is still `db_service` in the Teleport config file, right? As such this PR will work as-is and the role string used will just be outwardly inconsistent with `6.0.0-alpha.1` (but then right as of `6.0.0-alpha.2`)